src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -425,29 +440,18 @@ public static double getLocalTimezoneHourOffset() {
         return TimeZone.getDefault().getOffset(new Date().getTime()) / 1000.0 / 60.0 / 60.0;
     }
 
-    private static Date convertToDate(String date, int time) {
-        SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy");
-        sdf.setTimeZone(SystemParams.TIME_ZONE);
-        Calendar calendar = Calendar.getInstance(SystemParams.TIME_ZONE);
-
-        // Perform date manipulation
-        try {
-            Date newDate = sdf.parse(date);
-            calendar.setTime(newDate);
-
-            if (time == 2400) {
-                calendar.set(Calendar.HOUR, 23);
-                calendar.set(Calendar.MINUTE, 59);
-            } else {
-                calendar.set(Calendar.HOUR, time / 100);
-                calendar.set(Calendar.MINUTE, time % 100);
-            }
-
-            return calendar.getTime();
-        } catch (Exception e) {
-            return null;
-        }
+    /**
+     * Temporary method for transition from storing time zone as double.
+     */
+    public static ZoneId convertToZoneId(double timeZone) {
+        return ZoneId.ofOffset("UTC", ZoneOffset.ofTotalSeconds((int) (timeZone * 60 * 60)));
+    }
 
+    /**
+     * Temporary method for transition from java.util.Date.
+     */
+    public static Date convertLocalDateTimeToDate(LocalDateTime localDateTime) {
+        return localDateTime == null ? null : Date.from(localDateTime.atZone(ZoneId.of("UTC")).toInstant());
```

Comment:
I will just write a comment. This is a temporary method so I don't think we need a javadoc?

Reply:
You could just write `@param localDateTime the local datetime in UTC` and it'd be more convenient since intellij will show it in the quick doc and we don't have to remember that it must be in UTC.

Reply:
Alternatively, rename `convertLocalDateTimeToDate` to `convertUtcLocalDateTimeToDate` for self-documenting code.

Reply:
I see your point. Ok

Reply:
But note that if you need to think about whether when using this method, the `localDateTime` is in UTC or not, then you are probably doing the wrong thing. Timezone conversion should use other methods. This method is for blindly wrapping a `LocalDateTime` returned in the new functions that use Java 8.